.gitignore

```diff
@@ -186,4 +186,5 @@ FakesAssemblies/
 # LightSwitch generated files
 GeneratedArtifacts/
 _Pvt_Extensions/
-ModelManifest.xml
\ No newline at end of file
+ModelManifest.xml
+/AggregateMetrics/Test
```

Comment:
this is smart. But it had still added the project into solution file. Merging as is, fix it later so VS wouldn't complain for a missing project